contracts/cw-core/src/contract.rs

```diff
@@ -706,9 +756,31 @@ pub fn query_cw20_balances(
 }
 
 #[cfg_attr(not(feature = "library"), entry_point)]
-pub fn migrate(_deps: DepsMut, _env: Env, _msg: MigrateMsg) -> Result<Response, ContractError> {
-    // Don't do any state migrations.
-    Ok(Response::default())
+pub fn migrate(deps: DepsMut, _env: Env, msg: MigrateMsg) -> Result<Response, ContractError> {
+    match msg {
+        MigrateMsg::FromBeta {} => {
```

Comment:
How would you feel about calling this variant `FromV1` as that's technically what we're migrating from?

Reply:
Bumping this. :)

Reply:
sorry! missed this